platforms/jvm/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/jupiter/JUnitJupiterParameterizedClassIntegrationTest.groovy

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.testing.junit.jupiter
+
+import org.gradle.api.tasks.testing.TestResult
+import org.gradle.testing.junit.platform.JUnitPlatformIntegrationSpec
+
+import static org.hamcrest.Matchers.equalTo
+
+class JUnitJupiterParameterizedClassIntegrationTest extends JUnitPlatformIntegrationSpec {
+    def "parameterized class has reasonable hierarchy"() {
```

Comment:
💭 Might also be worth adding a parameterized test to this parameterized class, just to see.